extensions/2.0/Khronos/KHR_technique_webgl/README.md

```diff
@@ -310,7 +236,8 @@ Shader source files are stored in the asset's `shaders` dictionary property, whi
     ]
 }
 ```    
-### Properties Reference
+
+## Properties Reference
```

Comment:
In this section, `.programs`, `.shaders`, and `.techniques` are now arrays with indices rather than an "ID in the global glTF namespace".